src/pages/__tests__/Processes.test.tsx

```diff
@@ -69,4 +75,87 @@ describe("Processes page", () => {
   it("renders correctly", () => {
     expect(screen).toMatchSnapshot();
   });
+
+  it("shows text content correctly", async () => {
+    expect(await screen.findAllByText("Processos")).not.toBe(null);
+    expect(await screen.findByText("Registro")).not.toBe(null);
+    expect(await screen.findByText("Apelido")).not.toBe(null);
+    expect(await screen.findByText("Situação atual")).not.toBe(null);
+    expect(await screen.findByText("Fluxo")).not.toBe(null);
+    expect(await screen.findByText("Status")).not.toBe(null);
+    expect(await screen.findByText("Ações")).not.toBe(null);
+  });
+
+  it("shows process text content correctly", async () => {
+    expect(await screen.findByText("12345678912345678915")).not.toBe(null);
+    expect(await screen.findByText("Processo não Iniciado")).not.toBe(null);
+    expect(await screen.findByText("Não iniciado")).not.toBe(null);
+    expect(await screen.findByText("12345678912345678916")).not.toBe(null);
+    expect(await screen.findByText("Processo Arquivado")).not.toBe(null);
+    expect(await screen.findByText("Arquivado")).not.toBe(null);
+    expect(await screen.findByText("12345678912345678917")).not.toBe(null);
+    expect(await screen.findByText("Processo em Andamento")).not.toBe(null);
+    expect(await screen.findByText("Em andamento")).not.toBe(null);
+    expect(await screen.findByText("12345678912345678918")).not.toBe(null);
+    expect(await screen.findByText("Processo Finalizado")).not.toBe(null);
+    expect(await screen.findByText("Finalizado")).not.toBe(null);
+  });
+
+  it("shows 'create process' correctly", async () => {
+    const button = screen.getByText("Criar Processo");
+
+    if (button) {
+      expect(button).not.toBe(null);
+    }
+  });
+
+  it("shows 'legal priority' correctly", async () => {
+    const button = screen.getByText(
+      "Mostrar apenas processos com prioridade legal"
+    );
+
+    if (button) {
+      expect(button).not.toBe(null);
+    }
+  });
+
+  it("shows 'archived/finished' correctly", async () => {
+    const button = screen.getByText(
+      "Mostrar apenas processos arquivados/finalizados"
+    );
+
+    if (button) {
+      expect(button).not.toBe(null);
+    }
+  });
+
+  it("shows 'search bar' correctly", async () => {
+    const button = screen.getByPlaceholderText(
+      "Pesquisar processos (por registro ou apelido)"
+    );
+
+    if (button) {
```

Comment:
galera, tirem esses if's, se voces so testarem se o botão aparece toda vez que button tiver valor, esse teste vai passar sempre, pq se o button nao for achado ele nao entra na linha de baixo que testa justamente isso

Reply:
Entendi, no caso ainda preciso remover? pe no PR que testa o toggle de arquivados/finalizados isso já foi corrigido

Reply:
remove antes por favor, acredito que merjando isso seria sobrescrito